README.md

```diff
@@ -29,7 +29,8 @@ Fee oracle needs three config files in the `./` dir of the codebase:
 ### Application config file `config.yaml`
 Template of the config.yaml can be found in `./config/config.template.yaml`.
 
-### Domain config file `domain.json`
+### Domain config
+For domain configuration, it is posible to use local file, or the [shared configuration](https://github.com/sygmaprotocol/sygma-shared-configuration) file that is uploaded to some remote service (eg. ipfs). Depending on what value is provided to the flag `domain_config_path`, the aplication will use the domain configuration from local file or from the remote service.
```

Comment:
Can you just explicitly note that depending **if the URL or file path** is provided to `domain_config_path`, it will load...